herringbone-main/src/main/scala/com/stripe/herringbone/CompactInputFormat.scala

```diff
@@ -22,9 +22,8 @@ import parquet.example.data.simple.SimpleGroup
 
 class CompactInputFormat[T](readSupportClass: Class[_ <: ReadSupport[T]]) extends ParquetInputFormat[T](readSupportClass) {
 
-  // We can't accurately predict the size of the resulting merged file, so aim
-  // for 900MB. Our HDFS block size is 1024MB so we'll get pretty close.
-  val TARGET = 1024 * 1024 * 900 // 900MB.
+  // Our HDFS block size is 1024MB so we'll get pretty close.
+  val TARGET = 1024 * 1024 * 1024 // 1024MB.
```

Comment:
To match another internal tool that we're going to merge into this one, honestly. This fix is a move towards consistency as well. ^^ I'll fix the comment below!